tests/core/policies/test_rule_policy.py

```diff
@@ -93,8 +93,37 @@ def _form_activation_rule(
     )
 
 
+def test_restrict_multiple_user_inputs_in_rules():
+    domain = Domain.from_yaml(
+        f"""
+intents:
+- {GREET_INTENT_NAME}
+actions:
+- {UTTER_GREET_ACTION}
+    """
+    )
+    policy = RulePolicy(restrict_rules=True)
+    greet_events = [
+        UserUttered(intent={"name": GREET_INTENT_NAME}),
+        ActionExecuted(UTTER_GREET_ACTION),
+        ActionExecuted(ACTION_LISTEN_NAME),
+    ]
```

Comment:
let's re-use the events from `GREET_RULE`

Reply:
but it contain rule snippet action, which will need to be excluded, not sure whether it'll make it easier?